addons/kommander/1.x/kommander-11.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kommander
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-10"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-11"
     appversion.kubeaddons.mesosphere.io/kommander: "1.0.0"
-    endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander/ui
+    endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander/ui/
```

Comment:
:thinking: did this cause a problem somewhere?

Reply:
Yeah the UI doesn't handle with no trailing slash and it doesn't redirect to the include the trailing slash. We plan on fix that tho

Reply:
:+1: